src/content/tutorials/build-a-slackbot/index.md

```diff
@@ -0,0 +1,696 @@
+---
+updated: 2020-06-29
+difficulty: Beginner
+length: 100%
+---
+
+# Build a Slackbot
+
+In this tutorial, you’ll build a [Slack](https://slackhq.com) bot using [Cloudflare Workers](https://workers.cloudflare.com). Your bot will make use of GitHub webhooks to send messages to a Slack channel when issues are updated or created, and allow users to write a command to look up GitHub issues from inside Slack.
+
+![Build a Slack Application](./media/issue-command.png)
```

Comment:
Let’s be more descriptive here. This is the image:

![Slack chat Workers Bot message example](https://user-images.githubusercontent.com/154613/86294355-5c9c0600-bbc2-11ea-827c-d3f9f3adda6f.png)

Maybe something like “Slack chat Workers Bot message example”?